src/plugins/commit-message/index.js

```diff
@@ -45,10 +97,41 @@ function checkCommitMessage(message) {
         // If no suffix, or issue ref occurs before suffix, message is invalid
         if (!issueSuffixMatch || POTENTIAL_ISSUE_REF_REGEX.test(commitTitle.slice(0, issueSuffixMatch.index))) {
             isValid = false;
+            errCode.push("WRONG_REF");
         }
     }
 
-    return isValid;
+    return [isValid, errCode];
+}
+
+/**
+ * Create a comment message body with the error details
+ * @param {Array<string>} errCodes - list of the error codes
+ * @param {boolean} isTitle - whether it is for title or the commit message
+ * @param {string} username - username of the PR author
+ * @returns {string} the message to comment
+ * @private
+ */
+function commentMessage(errCodes = [], isTitle = false, username) {
+    const errorMessages = [];
+
+    errCodes.forEach(err => {
+        if (ERR_CODES_MAPS[err]) {
+            errorMessages.push(ERR_CODES_MAPS[err]);
+        }
+    });
+
+    return `Hi @${username}!, thanks for the Pull Request
+
+The ${isTitle ? "PR title" : "first commit message"} format is not according to our format.
```

Comment:
```suggestion
The ${isTitle ? "pull request title" : "first commit message"} isn't properly formatted.
```

Reply:
Can we explain why we ask contributors to format commit messages this way? 
```suggestion
The ${isTitle ? "pull request title" : "first commit message"} isn't properly formatted. We ask that you update the message to match this format, as we use it to generate changelogs and automate releases.
```